src/PayloadParser.h

```diff
@@ -394,7 +394,7 @@ namespace snowcrash {
                                    const ParseResultRef<Payload>& out) {
 
             const char* regex;
-            mdp::ByteBuffer mediaType;
+	    mdp::ByteBuffer mediaType;
```

Comment:
Seems like a github bug, look at the code https://github.com/apiaryio/snowcrash/blob/kylef/request-syntax/src/PayloadParser.h#L397